src/benchmarklib/synthetic_table_generator.cpp

```diff
@@ -50,28 +50,22 @@ namespace opossum {
 std::shared_ptr<Table> SyntheticTableGenerator::generate_table(const size_t num_columns, const size_t num_rows,
                                                                const ChunkOffset chunk_size,
                                                                const SegmentEncodingSpec segment_encoding_spec) {
-  auto table =
-      generate_table({num_columns, {ColumnDataDistribution::make_uniform_config(0.0, _max_different_value)}},
-                     {num_columns, {DataType::Int}}, num_rows, chunk_size,
-                     std::vector<SegmentEncodingSpec>(num_columns, segment_encoding_spec), std::nullopt, UseMvcc::No);
+  ColumnSpecification column_specification = {{ColumnDataDistribution::make_uniform_config(0.0, _max_different_value)},
```

Comment:
The last two std::nullopt could be omitted here

Reply:
Resolved in 9425a461a6a5665302b3e1bf26d383b825b0ffc8.